### ISSUE TITLE
What is the IP address of my raspberry pi if I use 10.0.X.X

### ISSUE THREAD

**Issue by viorizz** [Ysurac/openmptcprouter#3415]
## Expected Behavior

Well the raspberry should be accessible using 192.168.100.1

## Current Behavior

Since I don't use this ip range but I use 10.0.X.X I can't access it. Or atleast I don't think I can, because when I try to access it by typing 10.0.100.1 it doesn't work.

My current IP range is set to 10.0.0.101 - 10.0.0.220

## Specifications

  - OpenMPTCProuter version: v0.60-6.1
  - OpenMPTCProuter VPS version: Haven't yet installed it
  - OpenMPTCProuter VPS provider: Hostinger
  - OpenMPTCProuter platform: RPI4B

**Comment by viorizz** [comment]
Also might be interesting to add that I scanned my network with 3 different tools and still can't get anything the raspberry doesn't show up nowhere. I made a fresh install on the RPI, it's brand new received it today. Both leds are constant, and the network leds are blinking. I'm wondering if it's something I did wrong. Mabye I installed the RPI wrong. Did I need to install Raspberry os and on top install OMR ? I don't really know. But I could really use some advice here.
